src/app/container/paramfiles/paramfiles.service.ts

```diff
@@ -41,7 +41,7 @@ export class ParamfilesService {
       return this.workflowsService.getTestParameterFiles1(id, versionName);
     } else {
       if (descriptor === 'CWL' || descriptor === 'WDL') {
-        return this.containersService.getTestParameterFiles(id, versionName, descriptor);
+        return this.containersService.getTestParameterFiles(id, descriptor, versionName);
```

Comment:
Think these changes are a result of the descriptor argument becoming a required field in the openapi.yaml: 
https://github.com/dockstore/dockstore/commit/f03a1b4a5ad4d1abf03e7b2ebc247d0ee1877598#diff-575c3ab71bc5c00666e643068d89643a17158e9ad91671590043a462d05bc7e9

Reply:
Ouch, I guess the parameters that can be null go last... Or it's random.